TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/LinearSlideSubsystem.java

```diff
@@ -55,17 +55,13 @@ public LinearSlideSubsystem(OpMode opMode) {
 
     public void runPIDPosition() {
 
-        /*if(motor.getCurrentPosition() > targetPosition - 5 && motor.getCurrentPosition() < targetPosition + 5) {
-            motor.setPower(0);
-            return;
-        }*/
         double outputPower = controller.calculate(motor.getCurrentPosition(), targetPosition);
         telemetry.addData("Power to motors:", outputPower);
 
 
         if(Math.abs(outputPower) > 0.6 ) {
```

Comment:
You should leave a comment about why it's 0.6 and this value should probably been in Constants.

Reply:
This is just a random number by Brandon, never tested it if it is effective or not (but the linear slide worked fine after all)